src/psyclone/tests/psyir/frontend/fparser2_subroutine_handler_test.py

```diff
@@ -207,11 +207,11 @@ def test_function_type_prefix(fortran_reader, fortran_writer,
         f"  function my_fUnc()\n"
         f"    {basic_type} :: my_fUnc\n"
         f"\n"
-        f"    my_fUnc = {rhs_val}\n"
+        f"    my_func = {rhs_val}\n"
```

Comment:
Just to double check, are the changes in this test still necessary?

Reply:
No, I don't think they are.